sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/tests/Azure.Monitor.OpenTelemetry.Exporter.Tests/MetricHelperTests.cs

```diff
@@ -43,7 +43,7 @@ public void ValidateMetricTelemetryItem()
             Assert.Equal(2, metricsData.Version);
             Assert.Equal("TestDoubleCounter", metricsData.Metrics.First().Name);
             Assert.Equal(123.45, metricsData.Metrics.First().Value);
-            Assert.Equal(DataPointType.Aggregation, metricsData.Metrics.First().DataPointType);
+            Assert.Null(metricsData.Metrics.First().DataPointType);
```

Comment:
Why asset on Null?

Reply:
I was going to ask the same question. Did we lose something with this refactor?

Reply:
Removing `DataPointType` is intentional. I think asserting on it does not hurt. @TimothyMothra - not sure what you mean?

Reply:
this PR is not a refactor! it is intentionally removing DataPointType. (and doing refactoring too!)

Reply:
Sorry, ignore my comment, I missed that DataPointType was intentionally removed.